style.css

```diff
@@ -4,23 +4,12 @@
     font-family: sans-serif;
 }
 
-body {
-    display: flex;
-    flex-direction: column;
-    gap: 1px;
-    height: 1080px;
```

Comment:
Avoid using fixed dimensions like a height in layout styes. Fixed dimensions can make the layout brittle and less responsive.